commands/deploy.go

```diff
@@ -158,13 +162,15 @@ func (c *DeployCmd) Run() (err error) {
 	}
 	s.Suffix = " Deploying app..."
 	s.Start()
-	serviceURL := c.ApertureURL + fmt.Sprintf(c.EnvUpdatePathFmt, c.AccountID, c.AppID, "production")
-	err = triggerUpdate(c, response.PayloadID, serviceURL, client)
+	up := api.EnvironmentUpdateCommand{
+		Op: "replace",
+		Value: PayloadValue{
+			ID: response.PayloadID,
+		},
+	}
+	err = api.ApplicationEnvironmentModuleUpdate(c.AccountID, c.AppID, "production", "nodejs/.section-external-source.json", up)
```

Comment:
Not asking to change here but just wondering, is "production" akin to the branch name? Does this work for one of the "staging" environments? Maybe we never make those anyways with the NodeJS app flow, but wondering if there is future work we should record.

Reply:
That's right, `production` is the branch name, so this will only work for that environment. 

I have a todo to add support for multiple environments, which I have recorded as #45.